framework/tests/all-systems/Issue691.java

```diff
@@ -0,0 +1,9 @@
+// Test case for Issue 691:
+// https://github.com/typetools/checker-framework/issues/691
+
+interface MyInterface<T> {
+}
+@SuppressWarnings("")
```

Comment:
It would be helpful adding a commit about why `@SuppressWarnings("")` is used.  And that the code below cause GLB to be executed, which results in a crash if the default implementation isn't correct for a given checker.